lib/node_modules/@stdlib/math/base/special/minmax/lib/main.js

```diff
@@ -20,75 +20,36 @@
 
 // MODULES //
 
-var compute = require( './minmax.js' );
+var compute = require( './assign.js' );
 
 
 // MAIN //
 
 /**
 * Returns the minimum and maximum values.
 *
-* @param {(Array|TypedArray|Object)} [out] - output object
 * @param {number} x - first number
 * @param {number} [y] - second number
 * @param {...number} [args] - numbers
-* @returns {(Array|TypedArray|Object)} minimum and maximum values
+* @returns {Collection} minimum and maximum values
 *
 * @example
 * var v = minmax( 3.14, 4.2 );
 * // returns [ 3.14, 4.2 ]
 *
 * @example
-* var out = [ 0.0, 0.0 ];
-* var v = minmax( out, 5.9, 3.14, 4.2 );
-* // returns [ 3.14, 5.9 ]
-*
-* var bool = ( v === out );
-* // returns true
-*
-* @example
 * var v = minmax( 3.14, NaN );
 * // returns [ NaN, NaN ]
 *
 * @example
 * var v = minmax( +0.0, -0.0 );
 * // returns [ -0.0, 0.0 ]
 */
-function minmax( out, x, y ) {
-	var bool;
+function minmax( x, y ) { // eslint-disable-line no-unused-vars
 	var args;
-	var len;
-	var i;
 
-	len = arguments.length;
-	if ( len === 1 ) {
-		return compute( [ 0.0, 0.0 ], out );
-	}
-	bool = ( typeof arguments[ 0 ] === 'number' );
-	if ( len === 2 ) {
-		if ( bool ) {
-			return compute( [ 0.0, 0.0 ], out, x );
-		}
-		return compute( out, x );
-	}
-	if ( len === 3 ) {
-		if ( bool ) {
-			return compute( [ 0.0, 0.0 ], out, x, y );
-		}
-		return compute( out, x, y );
-	}
-	if ( bool ) {
-		args = [];
-		args.push( [ 0.0, 0.0 ] );
-		i = 0;
-	} else {
-		args = [];
-		args.push( arguments[ 0 ] );
-		i = 1;
-	}
-	for ( ; i < len; i++ ) {
-		args.push( arguments[ i ] );
-	}
+	args = [].slice.call( arguments, 0 );
```

Comment:
We need to explicitly copy arguments using a `for` loop. Passing `arguments` to a function can result in an automatic de-opt in older JS engines.